workspaces/ui-v2/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM library/nginx:alpine
+ARG APP
+COPY build-${APP}/ /usr/share/nginx/html/
+COPY config/nginx/ /etc/nginx/
```

Comment:
we add an nginx config (https://github.com/opticdev/optic/tree/develop/workspaces/ui/config/nginx) that sets security headers. it should largely just a drop-in, but give the CSP header a read in case there's anything to add or remove.

Reply:
Pushed!